application/backend/src/main/resources/apps/typerefinery/components/structure/page/clientlibs/functions.js

```diff
@@ -4,89 +4,121 @@ window.Typerefinery.Theme = Typerefinery.Theme || {};
 window.MessageService = window.MessageService || {};
 window.MessageService.Client = MessageService.Client || {};
 
-; (function (ns, themeNs, clientNs, componentNs, document, window) {
-    
-    "use strict";
-    ns.handleTheme = () => {
-        const currentTheme = localStorage.getItem('theme');
-        const style = document.getElementById('themeStyles');
-        if (!currentTheme) {
-            localStorage.setItem('theme', 'dark');
-        }
-        if (currentTheme === 'light') {
-            style.setAttribute('active', 'light');
-        } else {
-            style.setAttribute('active', 'dark');
-            style.setAttribute('href', `/apps/typerefinery/web_root/dark.css`);
-        }
-    };
-    ns.hostAdded = (newHost) => {
-        const listOfHost = JSON.parse(localStorage.getItem("tmsHost") || '');
-        const filteredHost = listOfHost.filter(host => host === newHost);
-        if(filteredHost.length === 0) {
-            listOfHost.push(newHost);
-            localStorage.setItem(JSON.parse(listOfHost));
-        }
+(function (ns, themeNs, clientNs, componentNs, document, window) {
```

Comment:
why there is a change in the whole file?

Please format the code which you wrote.

Reply:
Using prettier ,it is formatted, so let it be like this

Reply:
I cant see what changes you made, please undo them and format your code.